packages/quick_actions/lib/quick_actions.dart

```diff
@@ -7,9 +7,6 @@ import 'dart:async';
 import 'package:flutter/services.dart';
 import 'package:meta/meta.dart';
 
-const MethodChannel _kChannel =
```

Comment:
Can we still keep this and use this const when we init the QuickAction using default?
Something like 
```dart
static final QuickActions _instance = QuickActions.private(
    const MethodChannel(_kChannel),
  );
```